paper/mel_report.Rmd

```diff
@@ -31,37 +34,237 @@ knitr::opts_chunk$set(fig.align = 'center',
+
+*The attitude domain has four sub-domains: general attitudes toward GBV, GBV unacceptability, attitudes towards gender roles, and attitudes toward professional role in responding to GBV. The attitudes toward professional role in responding to GBV sub-domain was included in outcome 5 rather than outcome 4.
+
+True/false and yes/no questions were scored 1 for correct answers and 0 for incorrect, unanswered questions, or responses of “I don’t know.” Questions using a Likert scale were scored as the number of responses minus 1, with the least desirable response being scored as 0. 
+
+For the empathy and attitude domains only, questions were worded both positively and negatively. Responses were scored according to the following scale for positively worded items (for which preferred answer was “always”): never = 0; rarely = 1; sometimes = 2; often = 3; always = 4. For negatively worded items (for which preferred answer was “never”), the scores were reversed to account for the desired response.
 
+Composite variables for each domain and subdomain were created by summing the scores of each survey item within each the respective domain. All domains’ scores were normalized on a 100-point scale. 
+
+Given the small sample size, CHCs and health posts were collapsed into one "CHC catchment area."
+
+**Self-reported practices**
+Participants self-reported practices in identifying and caring for women subjected to GBV in the survey. The proportion of providers who identified a woman suffering from GBV among their female patients in the past month was calculated. Among those providers who identified women suffering from GBV, the average number of patients identified and the proportion of appropriate actions taken were calculated at baseline and endline.
+
+# Results
+## Demographic information
+Matched data was available for 46 individuals (138 surveys) from a total of 676 surveys over the three timepoints. Participant demographic characteristics and work history at baseline are presented in Table 2.
+
+The overall sample was 63% female (29 participants) and 37% male (17 participants). 50% of the sample was between 25-34 years old (22 participants). The most common position held was medical doctor (32% or 14 participants), followed by nurse (27% or 12 participants) and midwife (25% or 11 participants). At baseline, participants had been practicing in their clinical roles for an average of 7 years. Most participants saw an average of 39 or fewer patients per week, with 27% (12 participants) seeing less than 20 patients per week and another 27% seeing between 20-39 patients per week. 24% of participants (11 participants) saw 60 or more patients per week. Only 8.7% of participants (4 participants) had ever received GBV training before.
```

Comment:
For the numbers, you can actually take those from the actual table in the text directly. That way, if the table changes, you don't have to update the text. Here is an example: 
The overall sample was `r inline_text(demographic_table, variable = sex_factored, level = "Female", column = "stat_0")` female and `inline_text(demographic_table, variable = sex_factored, level = "Male", column = "stat_0")` male. `inline_text(demographic_table, variable = age_groups, level = "25-34 years old", column = "stat_0")` of the sample was between 25-34 years old.

That will write out the stats directly in the word document.